label_studio/projects/models.py

```diff
@@ -997,23 +998,30 @@ def get_active_ml_backends(self):
 
         return self.get_ml_backends(state=MLBackendState.CONNECTED)
 
-    def get_all_storage_objects(self, type_='import'):
+    @cached_property
+    def get_all_import_storage_objects(self):
         from io_storages.models import get_storage_classes
 
-        if hasattr(self, '_storage_objects'):
-            return self._storage_objects
+        storage_objects = []
+        for storage_class in get_storage_classes('import'):
+            storage_objects += list(storage_class.objects.filter(project=self))
+
+        return storage_objects
+
+    @cached_property
+    def get_all_export_storage_objects(self):
+        from io_storages.models import get_storage_classes
 
         storage_objects = []
-        for storage_class in get_storage_classes(type_):
+        for storage_class in get_storage_classes('export'):
             storage_objects += list(storage_class.objects.filter(project=self))
 
-        self._storage_objects = storage_objects
```

Comment:
was this the issue? We would save import storages here, then above 
```
if hasattr(self, '_storage_objects'):
```
We would return import storages even though we requested export?

Reply:
yep, that's my read, we just don't request export anywhere